s3-source-connector/src/integration-test/java/io/aiven/kafka/connect/s3/source/IntegrationTest.java

```diff
@@ -337,13 +339,18 @@ private static byte[] generateNextAvroMessagesStartingFromId(final int messageId
     }
 
     private static String writeToS3(final String topicName, final byte[] testDataBytes, final String partitionId) {
-        final String objectKey = topicName + "-" + partitionId + "-" + System.currentTimeMillis() + ".txt";
+        final String objectKey = addPrefixOrDefault("") + topicName + "-" + partitionId + "-"
```

Comment:
can we add a prefix to any one of the integration tests and assert for count ?

Reply:
The prefix was previously added in the config for the integration tests, but not being used. We are actually using the prefix correctly in our integration tests now.